### PR TITLE
Add cloud-native software supply chain security definition

### LINKED ISSUE #461: Add paragraph about what "cloud native" software supply chain security entails to Software Supply Chain Security article

### PR DESCRIPTION
**Pull request by jspeed-meyers**
### What should this PR do?

Adds an explanation of cloud-native software supply chain security to the academy website.

### Why are we making this change?

Fix #461 

### What are the acceptance criteria? 

@sheesh should read it and review it.

### How should this PR be tested?

Make sure the web page still renders properly.

### CODE REVIEW

File: content/software-security/what-is-software-supply-chain-security.md [hunk 1]
```diff
@@ -27,3 +27,4 @@ Malicious attacks, or what often amounts to code tampering, deserve special reco
...
 
+One further term merits a definition and explanation. Cloud-native software software supply chain security refers to software supply chain security efforts that are related to container technology. The process of selecting, building, and operating containers has a number of important implications for software supply chain security. For instance, signing containers with a digital signature (via a tool like [Cosign](https://github.com/sigstore/cosign)) is one way to ensure that an attacker that tampers with a container is detected. Container technology both epitomizes the potential software supply chain security perils that modern organizations must confront while also enabling approaches that can make a software supply chain secure by default.
```
Comment (ltagliaferri) [inline content/software-security/what-is-software-supply-chain-security.md:30]:
```suggestion
One further term merits a definition and explanation. Cloud-native software supply chain security refers to software supply chain security efforts that are related to container technology. The process of selecting, building, and operating containers has a number of important implications for software supply chain security. For instance, signing containers with a digital signature (via a tool like [Cosign](https://github.com/sigstore/cosign)) is one way to ensure that an attacker that tampers with a container is detected. Container technology both epitomizes the potential software supply chain security perils that modern organizations must confront while also enabling approaches that can make a software supply chain secure by default.
```

**Review by ltagliaferri — APPROVED** [review]
One copyedit, thank you 😄

**Comment by jspeed-meyers** [comment]
Thank you, @ltagliaferri!

**Comment by sheesh** [comment]
Looks great! Thanks for adding, @jspeed-meyers!

**Pull request merged**